airport/airport-pojo-bl/src/main/java/berlin/assets/Asset.java

```diff
@@ -43,25 +42,23 @@
     @Title(value = "Number", desc = "A unique asset number, auto-generated.")
     @CompositeKeyMember(1)
     @Readonly
-    private String number;
-
+    private String number;    
+    
     @IsProperty
     @MapTo
-    @Title(value = "Condition Rating", desc = "Condition rating for an asset")
-    private ConditionRating conditionRating;
+    @Title(value = "Asset Type", desc = "Asset Type of an Asset")
```

Comment:
Strictly speaking the description here is not needed as it does not bring much value. So, `@Title("Asset Type")` would be sufficient. Further more, due to the fact that this title is really just the title for entity `AssetType`, it can be removed all together -- try removing it and write a unit test that checks the title for this property by asserting the value of `asset.getProperty('assetType').getTitle()`.

Reply:
fixed in the latest commit https://github.com/SA-TeamBerlin/airport/pull/30/commits/c14d57232fcc401a3a0a1ab49d08124bfaa499d1